proto/clerk/v1beta/clerk.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package heimdall.clerk.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/maticnetwork/heimdall/x/clerk/types";
+
+option (gogoproto.marshaler_all)   = false;
+option (gogoproto.unmarshaler_all) = false;
+
+message EventRecord {
+    option (gogoproto.goproto_getters)  = false;
+
+    uint64 id = 1
+        [(gogoproto.jsontag) = "id", (gogoproto.moretags) = "yaml:\"id\""];
+    bytes contract = 2
```

Comment:
let's use sdk.AccAddress? 

Oh, may be we should use `string`

Reply:
Can't we use `sdk.AccAddress` or GRPC problem with it?

Reply:
@anilCSE

Reply:
@jdkanani let's keep strings for now. We can address it again if the hacky method looks good. I couldn't test it on the cosmos codebase, It required a full-refactoring

Reply:
got it.